Framework/DataHandling/src/LoadNexusProcessed.cpp

```diff
@@ -84,16 +83,16 @@ struct SpectraInfo {
   /*
   SpectraInfo& operator=(const SpectraInfo& other)
   {
-    if (&other != this)
-    {
-      nSpectra = other.nSpectra;
-      hasSpectra = other.hasSpectra;
-      spectraNumbers = other.spectraNumbers;
-      detectorIndex = other.detectorIndex;
-      detectorCount = other.detectorCount;
-      detectorList = other.detectorList;
-    }
-    return *this;
+  if (&other != this)
+  {
+  nSpectra = other.nSpectra;
+  hasSpectra = other.hasSpectra;
+  spectraNumbers = other.spectraNumbers;
+  detectorIndex = other.detectorIndex;
+  detectorCount = other.detectorCount;
+  detectorList = other.detectorList;
+  }
+  return *this;
   }
   */
```

Comment:
I see you only changed the formatting, but any idea why is the copy assignment operator commented out but not removed? Can also remove the copy constructor and rely on the implicitly-generated ones?